tools/lkl/lklfuse.c

```diff
@@ -71,6 +74,7 @@ static void usage(void)
 "    -o mb=memory           amount of memory to allocate in MB (default: 64)\n"
 "    -o part=parition       partition to mount\n"
 "    -o ro                  open file read-only\n"
```

Comment:
Please update -o ro to block device as well, for consistency with the change in the rst file